### PR TITLE
Survival kit for writing Conda's build.sh

### PR DESCRIPTION
**Pull request by abretaud**
Hey,
I've prepared these few slides for our small training next week (with @cmonjeau).

In previous sessions I sometimes saw despair and sadness in some students' eyes when the time came to write their first build.sh.

I think that's because they had no idea what are configure, make, make install, env vars, ...
So this time I'll try to explain them the minimum they should know to write a build.sh and understand why they get some errors.

We'll see if it helps them or not. (If not, well... I'll have to make some introspection on my training skills in a buddha temple (or accuse @cmonjeau))

### CODE REVIEW

**Comment by abretaud** [comment]
Thanks for the review, I made the changes.
I feel like the title is still a bit too long, but have no idea for a better one

File: topics/dev/tutorials/conda_sys/slides.html [hunk 1]
```diff
@@ -10,9 +10,9 @@
 ### Environment variables
 
 ```sh
-$ MY_NAME="Anthony"
+$ MY_NAME="Bobby"
 $ echo $MY_NAME
-Anthony
+Bobby
```
Comment (bgruening) [inline topics/dev/tutorials/conda_sys/slides.html]: New name? ;)

File: topics/dev/metadata.yaml [hunk 2]
```diff
@@ -275,7 +275,7 @@ material:
         type: "internal"
         link: "/topics/dev/tutorials/tool-integration/slides.html"
       -
-        title: "System administration prerequisites for Conda"
+        title: "System administration prerequisites for building conda packages"
```
Comment (bgruening) [inline topics/dev/metadata.yaml]: Maybe just `Prerequisites for building software/conda packages`

File: topics/dev/tutorials/conda_sys/slides.html (hunk 1 above)
Comment (abretaud) [reply topics/dev/tutorials/conda_sys/slides.html]: Yeah, this was getting too personal! (and bobby is so beautiful)

**Comment by bgruening** [comment]
Thanks @abretaud! Good luck with your workshop!

**Pull request merged**